From: Jakob Schiotz <schiotz@fysik.dtu.dk>
Date: Fri Aug 15 19:08:02 CEST 2025

Deselect parametrized test tests/cupyx_tests/scipy_tests/sparse_tests/test_dia.py::
TestDiaMatrixScipyComparison::test_sum_tuple_axis, which is parametrized and hard
to deselect with --deselect.

The test has been modified in the master branch due to incompatibility with SciPy 1.16.

--- tests/cupyx_tests/scipy_tests/sparse_tests/test_dia.py~	2025-07-10 14:59:19
+++ tests/cupyx_tests/scipy_tests/sparse_tests/test_dia.py	2025-08-15 19:05:29
@@ -245,6 +245,7 @@
         m = self.make(xp, sp, self.dtype)
         return m.A
 
+    @testing.with_requires('scipy<1.16')
     def test_sum_tuple_axis(self):
         for xp, sp in ((numpy, scipy.sparse), (cupy, sparse)):
             m = _make(xp, sp, self.dtype)
